### DISCUSSION TITLE
Link List

### DISCUSSION THREAD

**Discussion by Rozerinay** [nl-design-system/backlog#266] [Component suggestions]
<img width="480" alt="visuele weergave van het link list component" src="https://github.com/nl-design-system/backlog/assets/84851545/98fae84d-ee32-4c13-8087-00658b2a6748">

We willen graag meer leren over het Link List component. In welke vorm, of vormen, wordt deze momenteel ingezet en toegepast binnen jouw organisatie?

Deel hieronder een link naar Figma, Storybook, Github of screenshots. Met deze informatie kunnen we bewijzen dat het component onderdeel moet worden van de NL Design System component library.

Daarnaast wordt het delen van onderzoek gewaardeerd. Dus heb je zelf onderzoek gedaan en zijn er inzichten voor dit component? Dan horen we dit ook graag!

[Link naar bericht in Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1717662657908449)

---

## Naam
Link List

## Doel
Een lijst met gerelateerde links naar andere pagina’s of websites.

## Ook bekend als
Link Group

## Gerelateerde componenten
[Link](https://github.com/orgs/nl-design-system/discussions/206), [Unordered List](https://github.com/orgs/nl-design-system/discussions/229)

## Onderzoek
[Guidelines for Visualizing Links - NN Group](https://www.nngroup.com/articles/guidelines-for-visualizing-links/)
Artikel waarin wordt verteld dat het niet noodzakelijk is om Links in een lijst te onderstrepen.
Er is wel een uitzondering bij de Link kleuren rood en groen. Deze kleuren veroorzaken problemen voor gebruikers met veel voorkomende vormen van kleurenblindheid. In dit geval wordt geadviseerd om de Links wel te onderstrepen. 

[Link List color on intranets - NN Group](https://www.nngroup.com/articles/link-list-color-on-intranets/)
Artikel waarin wordt verteld dat het onderstrepen van Links in een grote lijst de leesbaarheid kan verminderen en, afhankelijk van de spatiëring en het lettertype, het rommelig kan ogen. 

[Formatting Links for Usability - Baymard Institute](https://baymard.com/blog/formatting-links-for-usability)
Artikel waarin wordt verteld dat het prima is om de onderstreping van Links te verwijderen die voorkomen in een lijst.

[Removing the external link icon from GOV.UK - GOV.UK](https://designnotes.blog.gov.uk/2016/11/28/removing-the-external-link-icon-from-gov-uk/)
Blogpost waarin wordt uitgelegd waarom de externe link icoon niet meer wordt gebruikt door gov.uk. 

[Onderzoek denhaag.nl - Gemeente Den Haag](https://miro.com/app/board/uXjVPYSokU0=/)
Miro bord waar een inzicht uit het gebruikersonderzoek staat over de externe Link icoon. Het is belangrijk dat de externe Link icoon niet wordt gebruikt wanneer er niet buiten de website genavigeerd wordt.

[Opmerking Bas over chevron in Link List - gemeente Amsterdam](https://github.com/orgs/nl-design-system/discussions/266#discussioncomment-9687934)
Bas vertelt dat in gebruikersonderzoek naar voren is gekomen dat de chevron de indruk wekte dat dit een uitklapbaar element is.

**Comment by Rozerinay** [comment]
## Amsterdam

- [Storybook](https://designsystem.amsterdam/?path=/docs/components-navigation-link-list--docs)
- [GitHub](https://github.com/Amsterdam/design-system/tree/develop/packages/css/src/components/link-list)
- [Figma](https://www.figma.com/design/9IGm6IdPUYizBNGsUnueBd/%F0%9F%8E%A8-Amsterdam-Design-System?m=auto&node-id=2927-29194&t=hE5sQEKvlKTkKRis-1)

---


### Screenshots / Varianten

**Storybook**

Default
Een verzameling van gerelateerde links.
<img width="376" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/4e3e4270-3fd0-4e52-a7c9-5036c3592953">

Custom icons
Een andere icoon dan de chevron in de list item wordt gebruikt om een link sneller of makkelijker te laten begrijpen.
<img width="388" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/10253ea0-87e6-4876-881a-896c6542785a">

Small text
In de footer wordt er small text gebruikt, ook voor lijsten.
<img width="380" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/a51f4cb3-db93-445b-b706-ba3864a26b61">

Link
Een item dat alleen voorkomt in een lijst en gerelateerd is aan één onderwerp, zoals: 'Alles over openbare orde en veiligheid', of voorkomt wanneer er geen andere items beschikbaar zijn.  
<img width="450" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/bc89e3a0-65bb-41a8-b49a-396babd1700f">

On Dark background
Lijst die wordt ingezet op donkere achtergronden, zoals in de footer.
<img width="450" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/3f5e3b8e-a27f-4055-a951-0bd2d08d5dbd">

On Light background
Lijst die wordt ingezet op lichte achtergronden.
<img width="450" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/fcfb0b3d-73d9-47fd-8ecf-5cd4aea8226b">


**Figma**
<img width="382" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/2dda4faf-31fa-4b7d-8798-908329f9b280">

<img width="531" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/19fb80f5-ad81-44a5-84fd-357ffc1beaef">

<img width="531" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/df4eac8e-6302-4750-98b4-f3054cfc97ff">

---

### Notities
- Optie voor icoon: vooraf aan het label.
- Icoon is aanklikbaar.
- Kan verschillende iconen bevatten. 
- Heeft bij default state geen underline, in tegenstelling tot de Link, die dit wel heeft. 
- Ook mogelijkheden voor een light of dark background.
- Heeft een andere styling op een gekleurde achtergrond.
- Kan ook één list item bevatten. 
- Richtlijnen beschikbaar (Storybook).

**Reply by bas-kroese** [reply]
@rozeq

Ja dit is inderdaad onze toepassing momenteel.

Nu is het wel zo dat er twijfel is onder designers of we het chevron zouden moeten gebruiken voor de link, omdat uit onderzoek met gebruikers is gebleken dat sommigen dit verwarren met een uitklapbaar element. Dat is de affondance die de chevron dan geeft.

Nog een voorbeeld hoe wij het toepassen, in de footer voor de socials:
![image](https://github.com/nl-design-system/backlog/assets/168745211/36f01d23-000c-460b-ae56-e31a57cc376e)

**Reply by Rozerinay** [reply]
@bas-kroese veel dank voor het delen! Heel waardevol.

Ik heb een aantal vragen over het onderzoek waar ik heel benieuwd naar ben:

- In welk context is de Link list onderzocht met gebruikers? Is het component getest waarbij één item aanwezig was of werd het in een lijst weergegeven?
- Weet je hoeveel gebruikers die verwarring ondervonden?
- Hebben jullie een vermoeden waardoor de verwarring ontstaat? Denk bijvoorbeeld aan de grootte van de icoon of de overeenkomsten met de Accordion (als de Accordion toevallig aan bod kwam in datzelfde onderzoek)?

**Reply by bas-kroese** [reply]
Terechte vragen. Ga ik achteran. Kom erop terug :)

**Reply by jeffreylauwers** [reply]
Oh ik lees dit toevallig net. Dit zijn mooie inzichten voor dit component en anderen waar chevrons (of pijllen) als iconen worden toegevoegd. Het gebruik van chevrons en pijlen is besproken tijdens het Design Open Hour 51. Waarbij we hebben uitgevraagd welke associatie designers (dus niet direct eindgebruikers!) hadden bij een combinatie van chevron of pijlen met tekst. Te vinden in het [Figma Schetsboek](https://www.figma.com/design/fy08SZpZmqx6ljLwvA3Woe/NLDS---Schetsboek?node-id=5084-5767).

**Comment by Rozerinay** [comment]
## Den Haag

- [Storybook](https://nl-design-system.github.io/denhaag/?path=/docs/css-linklist--docs)
- [GitHub](https://github.com/nl-design-system/denhaag/tree/main/components/LinkList)
- [Figma - Huidig](https://www.figma.com/design/JpoY3waVoQGlLQzQXTL9nn/🌻-Den-Haag-Design-System?node-id=1485-1156&t=KwYsF4w8XbBHPlrt-11)

---

### Screenshots / Varianten

**Storybook**

Default
Groep van links die de mogelijkheid biedt om naar andere pagina's te navigeren.
<img width="300" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/5076768b-c99a-4316-9932-bf2d743a51f8">

With image
Afbeelding die de lijst ondersteunt.
<img width="300" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/7a6f0e86-926e-44c4-bbe9-d5373297e135">

Dark background
Lijst die wordt ingezet op donkere achtergronden, zoals in de footer.
<img width="300" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/c65a7880-a251-40a8-b039-53261e0c94e6">

**Figma - Huidig**
<img width="500" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/0bb4bcf2-dd22-4bd9-a504-bc22ca821f65">

---

### Notities
- Onder de naam: Link group. 
- Optie voor icoon: vooraf aan het label en in design ook een icoon erachter.
- Icoon voor het label is altijd een arrow.
- Heeft bij default state geen underline, in tegenstelling tot de Link, die dit wel heeft.
- Ook mogelijkheden voor een light of dark background.
- Richtlijnen beschikbaar (Storybook).

**Comment by Rozerinay** [comment]
## DSO Toolkit

- [Documentatie website](https://www.dso-toolkit.nl/62.8.4/components/navigatie/link-list)
- [Storybook](https://storybook.dso-toolkit.nl/62.8.4/?path=/story/html-css-link-list--ul)
- [GitHub](https://github.com/dso-toolkit/dso-toolkit/tree/master/packages/dso-toolkit/src/components/link-list)

---


### Screenshots / Varianten

**ul en ol**
Groep van links die de mogelijkheid biedt om naar andere pagina's te navigeren of iets te downloaden.
<img width="200" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/b02e8c06-1a18-4f59-9d00-f988ac9da28d">

**In highlight box**
Lijst die wordt ingezet op gekleurde achtergronden, zoals in de footer.
<img width="400" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/fb70c13f-bfae-4651-ab4e-038ffa1c3d5e">

**In nav**
Lijst die wordt ingezet in de navigatie en de mogelijkheid biedt om te navigeren naar andere pagina's. Het laat tevens zien op welke pagina je je bevindt. 
<img width="300" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/bfe71d66-9bd3-4606-a79f-83f2baa8396f">

**In footer**
Lijst die wordt ingezet in de footer. Deze variant is ook te zien bij '[In highlight box](https://arc.net/l/quote/kixzxzzm)'.
<img width="200" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/fc904fb7-b088-4ba1-a737-127d61fc80d5">


---


### Notities

- Optie voor icoon: before en inline. De icoon after wordt overgenomen van het [Link](https://storybook.dso-toolkit.nl/62.8.4/?path=/story/voorbeeldpagina-s-toepassingen-componenten-link--link) component.  
- Icoon voor het label is altijd een chevron.
- Heeft bij default state geen underline, in tegenstelling tot de Link, die dit wel heeft.
- Ook mogelijkheden voor verschillende background colors. 
- Heeft een andere styling op een gekleurde achtergrond.

**Comment by Rozerinay** [comment]
## DUO

[Documentatie website](https://uno.dfront.rijkscloud.nl/#/components/lists)


---


### Screenshots / Varianten

**Link lijst**
Een reeks items die over een onderwerp gaan en de mogelijkheid biedt om naar verschillende pagina's te navigeren, zoals links onder [Meer informatie](https://arc.net/l/quote/uvdptnih). 
<img width="320" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/dec7a8d6-550b-46c3-b029-5eab73fd6f21">

**Link lijst met chevron**
Wordt gebruikt wanneer er een reeks items is met vergelijkbare informatie, zoals het tonen van alle nieuwsonderwerpen of alle pagina's rondom de [leerlingenadministratie](https://arc.net/l/quote/wttvqflb).
<img width="320" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/215c55d6-c0d3-4bb1-8384-823aec99b545">

**Link lijst met chevron op een achtergrond**
Lijst die wordt ingezet op gekleurde achtergronden.
<img width="320" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/32deece6-7ced-4f11-9690-619ae1cb5b42">

---

### Notities
- Beschikbaar onder List component.
- Optie voor icoon: voor en achter het label, maar ook een variant zonder icoon.
- Icoon voor het label is altijd een chevron en icoon achter het label een external link icon.
- Heeft bij default state een underline.
- Ook mogelijkheden voor een light of dark background.
- Richtlijnen beschikbaar (documentatie website).

**Comment by Rozerinay** [comment]
## OpenGemeenten

- [Documentatie website](https://componenten.lenteveld.nl/follow-up-organisatie/footer-contactlinks)
- [Patternlab](https://patternlab.lenteveld.nl/index.html)

---

### Screenshots / Varianten

**Documentatie website**

Footer contactlinks
Links in de footer die navigeren naar contactmogelijkheden zoals een telefoonnummer, e-mailadres of Whatsapp.
<img width="200" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/69dada6e-4271-463e-bc42-67c3f5c53aa2">

Footer socialmedialink
Links in de footer die navigeren naar social mediakanalen.
<img width="192" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/73d44f8e-c4b0-472d-935b-e9ad150b28da">

**Patternlab**

Metadata
<img width="600" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/a98c1888-54b7-487e-b554-6c62bbdfd373">

**dijkenwaard.nl**

Footer link list
Links in de footer die navigeren naar pagina's anders dan de social mediakanalen en contactmogelijkheden. 
<img width="200" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/abf53a0d-367f-4f4c-87cc-4551a370b5bd">

---

### Notities
- Footer contactlinks en Footer social media links bevatten een unordered list met daarin de class link-list.
- Lijst aan links is ook te zien onder het component Metadata.
- Optie voor icoon: voor het label.
- Kan verschillende iconen bevatten.
- Footer sociamedialink bevat enkel icoon en wordt horizontaal gepresenteerd.
- Ook mogelijkheden voor verschillende background colors.
- Heeft een andere styling op een gekleurde achtergrond.

**Comment by Rozerinay** [comment]
## Rotterdam

- [Storybook](https://nl-design-system.github.io/rotterdam/?path=/docs/css-utrecht-link-list--docs)
- [GitHub](https://github.com/nl-design-system/rotterdam/tree/main/rotterdam-nlds-parent-wicket/docs-wicket/src/main/java/nl/rotterdam/nl_design_system/docs/wicket/link_list)

---

### Screenshots / Varianten

**Storybook**

<img width="292" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/1201688b-6a84-47a5-b2ae-9fd22b5079f2">


**Rotterdam.nl**

List link
Wordt gebruikt wanneer er een reeks items is met vergelijkbare informatie over één onderwerp, zoals Belastingen.
Of wanneer verschillende soorten informatie wordt gepresenteerd, zoals wat je snel kunt regelen. 
<img width="400" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/1cb3fe6a-68de-45ac-81eb-18924a0aa703">

List item
<img width="400" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/00a251ef-f473-4fd9-8f37-801b59bca0ed">

<img width="400" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/5f85e5c6-6321-47dd-917f-0cc0b4a15b0d">

---

### Notities
- Op de website staan verschillende lijsten aan links als List item weergegeven. Eén daarvan is een List link.
- Verschil in code en design met betrekking tot de styling, padding en de icoon. 
- Optie voor icoon: voor het label.
- Heeft bij default state in Storybook geen underline en op de website wel een underline. 
- Heeft op de website een strong weight, in tegenstelling tot de Link, die een regular weight heeft.
- Ook mogelijkheden voor verschillende background colors.

**Comment by Rozerinay** [comment]
## RVO

- [Documentatie website](https://nl-design-system.github.io/rvo/docs/components/footer/)
- [Storybook](https://nl-design-system.github.io/rvo/?path=/docs/componenten-footer--documentatie)
- [GitHub](https://github.com/nl-design-system/rvo/tree/master/components/footer)
- [Figma](https://www.figma.com/design/Sj6myBL1Fvot5M1qGxzvEo/ROOS-(RVO-Design-System)?node-id=4467-15987&t=dClzJCkJbEzIlgpY-11)

---

### Screenshots / Varianten

**Documentatie website**

<img width="565" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/6325b737-1aef-43a7-bdc3-f45d50b17d2e">

**Storybook**

<img width="700" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/1197211c-ece6-4b5e-9d5d-99f8b03ebce7">


**Figma**

<img width="700" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/2aff06a0-ff47-46cc-9969-40275724a268">

**rvo.nl**
Footer item
<img width="1052" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/851ed108-943d-4bc3-ad5c-22743ea11b09">

Link list item
<img width="441" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/3b78f186-43bb-4949-b778-e21be26e8a96">


---

### Notities
- Beschikbaar onder Footer component als Footer menu item. 
- In design wordt de Button gebruikt. 
- In code wordt de Link gebruikt. 
- Optie voor icoon: voor het label. Op de website is een variant waarbij geen icoon wordt gebruikt. 
- Icoon voor het label is altijd een chevron.
- Heeft bij default state geen underline, in tegenstelling tot de Link, die dit wel heeft.
- Ook mogelijkheden voor een light of dark background.

**Comment by Rozerinay** [comment]
## Utrecht

- [Storybook](https://nl-design-system.github.io/utrecht/storybook/?path=/docs/css_css-link-list--docs)
- [GitHub](https://github.com/nl-design-system/utrecht/tree/main/components/link-list)

---

### Screenshots / Varianten

**Storybook**

Default
<img width="461" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/5cb53d8d-3302-41eb-9c65-019f30a8f862">

**utrecht.nl**

<img width="230" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/e21a3bb8-13c0-41d4-810e-74387c620c0c">

Andere weight
<img width="230" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/800f6027-311a-4a08-86f2-dfee232ab533">

Andere kleur
<img width="345" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/27b4f060-13cb-4db6-9444-53833f32f41d">

Eén list item
<img width="349" alt="image" src="https://github.com/nl-design-system/backlog/assets/84851545/78e5c1fd-a338-44cc-a55a-bec8f9676168">

---

### Notities

- Optie voor icoon: vooraf aan het label.
- Icoon voor het label is altijd een chevron.
- Heeft bij default state geen underline, in tegenstelling tot de Link, die dit wel heeft.
- Heeft een strong weight, in tegenstelling tot de Link, die een regular weight heeft.
- Op de website heeft de list item verschillende kleuren.
- Kan ook één list item bevatten.

**Comment by Rozerinay** [comment]
## ✨ Dit component is nu Help Wanted ✨

Help je mee hem door de Community stappen te krijgen?
[Link list op het Community bord](https://github.com/orgs/nl-design-system/projects/29/views/1?filterQuery=link+list)

**Comment by eslook** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Link list op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=link+list)

**Comment by TessaViergever** [comment]
De chevron icon in de Link List doet mij denken aan een expandable functionaliteit (en niet alleen een link). Wellicht dat een icon hier niet nodig is of gebruik van bijv. een arrow icon i.p.v. een chevron?

**Reply by jeffreylauwers** [reply]
Hey Tessa, interessant!

Heel eerlijk... ik kan me óók goed voorstellen dat eindgebruikers die verwachting hebben met de chevrons. Maar daar zou ik graag meer bewijs over verzamelen bijvoorbeeld door gebruikersonderzoek.

Ik heb een keer een mini-onderzoek gedaan naar de verwachting van 'chevrons' vs 'arrows' tijdens het Design Open Hour. Ik deelde mijn scherm. Liet per keer 1 afbeelding zien. Vroeg de aanwezigen designers om op het moment dat ik 'go' riep, hun antwoord in de chat te plaatsen. Daarbij moet ik natuurlijk opmerken dat het geen lijstjes betrof maar 1 icoon met 1 tekst ernaast.

Je vindt de antwoorden onder de afbeeldingen. Zoals je ziet loopt het nogal uiteen. 

Daar stond ik van te kijken trouwens. Zelf hanteer ik namelijk het volgende:

- Chevron: er klapt 'iets' open. De richting waar de chevron naar wijst geeft, indien mogelijk, aan op welke plek er 'iets' open gaat klappen.
- Arrow: biedt extra visuele ondersteuning om aan te geven dat je gaat navigeren. De richting waar de arrow's heeft ook nu weer een doen:
  - Rechts: Je navigeert verder.
  - Links: Je navigeert terug.
  - Beneden: Je navigeert binnen de pagina naar beneden (anker link)
  - Boven: Je navigeert binnen de pagina naar boven (back to top)

Maar zoals je in de voorbeelden in deze discussion ziet worden chevrons vaak eerder gebruikt dan arrows. Kortom we zullen eerst meer onderzoek moeten verzamelen of eindgebruikers ook de verwachting hebben dat er iets open klapt. Dus hierbij een oproep aan iedereen die dit leest. Mocht je dit tijdens je onderzoek vaker horen van eindgebruikers, deel dit dan vooral. Dat kan hier, of op https://gebruikersonderzoeken.nl/.

**Comment by jiromaykin** [comment]
In open-inwoner hebben we nog een ander component, dat we niet 'link list' hebben genoemd omdat het visueel meer lijkt op een verzameling Cards binnen een unordered list en het met meerdere iconen werkt.
Het gaat om het "Ga naar link" component dat gemeenten kunnen inzetten om door te linken naar andere externe platforms (zoals bijvoorbeeld financiële platforms zoals parkeervergunningen die nog niet in open-inwoner geïntegreerd zijn, of die een andere inlog vereisen).

Het icoon aan de linkerzijde is een custom icon dat gekozen kan worden, de pijl rechts is vast:

<img width="856" height="201" alt="ganaar" src="https://github.com/user-attachments/assets/e2c5ec3f-28d9-47cc-843c-ebdc5a44b16f" />

Code (HTML): https://github.com/maykinmedia/open-inwoner/pull/1996/files#diff-528d3b4a2ecb21e279c870cc028e40bbab265ebc56a4ad2a34099829b7b16053

design-tokens: https://github.com/maykinmedia/open-inwoner-design-tokens/pull/28/files

**Reply by jiromaykin** [reply]
Een uitgebreidere link list zoals in Open Inwoner is misschien meer een "Top Task Navigation", [zie aanvulling aldaar](https://github.com/orgs/nl-design-system/discussions/436#discussioncomment-16655019).

**Comment by julezrulez** [comment]
In de pagina die gaat over Link list wordt gesproken over "lijst". li-elementen worden opgemaakt met opsommingtekens. Een betere vertaling voor het gebruik van het ul-element is daarom opsomming, dus is er in goed Nederlands sprake van een "linksopsomming" of "opsomming van links".
